pkg/reconciler/sequence/sequence.go

```diff
@@ -215,6 +216,16 @@ func (r *Reconciler) reconcileChannel(ctx context.Context, channelName string, c
 		}
 		logging.FromContext(ctx).Error(fmt.Sprintf("Failed to get Channel: %s/%s", p.Namespace, channelName), zap.Error(err))
 		return nil, err
+	} else if !equality.Semantic.DeepEqual(newChannel.Object["spec"], c.Object["spec"]) {
```

Comment:
`subscribable` needs to be ignored as it is populated by subscriptions, not by the end-user.

Reply:
I revisit the code in generic channel, where the channelTemplate field is immutable. I think we should also let channelTemplate field in sequence immutable? Thus i remove the code to compare and update channel.

Reply:
yup sounds good too me.